docs/rollover.md

```diff
@@ -0,0 +1,72 @@
+# Rollover
+
+Each year we close the current cycles courses and open the new cycle in a
+process we call Rollover.
+
+This involves copying existing providers and courses to new records to allow the
+providers to update any details and then switching the API over to a new
+'recruitment cycle' which in turn releases the new courses on Find & Apply.
+
+In addition to this, providers also request places for courses that have numbers
+of applicants limited as they are generally over supplied.
+This process is called Allocations.
```

Comment:
```suggestion
This process is called 'Allocations'.
```

Reply:
I'm not sure is this really rollover or a different key point altogether.

Reply:
Good point - I'm working on the allocations documentation now, so will move it into there 👍